### PR TITLE
Updated Sidebar/Navbar Styling

### PR DESCRIPTION
**Pull request by RedbackThomson**
- Made the icons grayscale when inactive
- Added a hover background colour for all items
- Removed the top navbar on mobile
- Made sidebar new navbar and made it collapsible for mobile

### CODE REVIEW

**Comment by subhankar-panda** [comment]
this is what i get on a hard refresh - is nothing supposed to be on the sidebar?

image

**Comment by RedbackThomson** [comment]
There is meant to be a menu there, like there was before. I saw it, at least. Can you see if it is hidden?

File: src/client/layouts/components/admin/Sidebar.tsx [hunk 1]
```diff
@@ -1,55 +1,102 @@
...
+`
 
-const Link = styled(NavLink).attrs({
-  activeClassName
-})`
+const ListElements = styled.div<{
+  isOpen: boolean;
+}>`
+  ${props => !props.isOpen && css`
+    display: none;
```
Comment (subhankar-panda) [inline src/client/layouts/components/admin/Sidebar.tsx:34]: this seems to be triggering the items to not show. It makes sense - isOpen = false is the default init state

Comment (subhankar-panda) [reply src/client/layouts/components/admin/Sidebar.tsx:34]: is true meant to be the init state?

Comment (RedbackThomson) [reply src/client/layouts/components/admin/Sidebar.tsx:34]: Oh ummm I meant to add a media query to make sure it only hid them for mobile. Let me fix

**Comment by RedbackThomson** [comment]
kk try that now?

**Comment by subhankar-panda** [comment]
yay! thanks :)

**Pull request merged**